### ISSUE TITLE
Drop static db lookups for an API

### ISSUE THREAD

**Issue by jbrooksuk** [jbrooksuk/Intellitip#2]
It'd mean you'd have to be online, but at least we'd stay up to date.

**Comment by princemaple** [comment]
Would be good to keep the local version as a cache / backup. I'm in Australia, so if I have to hit an API endpoint in the U.S. it could potentially take 0.5 - 1s round trip, not to mention when the international cable is unstable.

**Comment by jbrooksuk** [comment]
If there was an API for language documentations then we'd cache it anyway, but there doesn't seem to be anything like it.

**Comment by princemaple** [comment]
Right.. Looks like another open source project opportunity. :smile:

**Comment by jbrooksuk** [comment]
There is [DevDocs.io](http://devdocs.io), but it doesn't have an API interface, which is why there is a script to update the database.

**Comment by princemaple** [comment]
Yup that's what I meant, either extend that project or build a wrapper
project.. Just an idea.

On Thu, Feb 19, 2015 at 1:23 AM, James Brooks <notifications@github.com>
wrote:

> There is DevDocs.io <http://devdocs.io>, but it doesn't have an API
> interface, which is why there is a script to update the database.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jbrooksuk/Intellitip/issues/2#issuecomment-74871053>.
>



-- 
Regards,

Po Chen
